### PR TITLE
test_: unskip test initialize logging

### PR DESCRIPTION
**Pull request by igor-sirotin**
1. Added `load_statusgo_data` that copies requested data from `status-backend` container to a temporary host path.
2. Unskipped `TestInitializeLogging` and updated it to use `load_statusgo_data`
3. `status-go` will no automatically create `DataDir` and `LogsDir` (if they don't exist) during `InitializeApplication`.
4. Switched to milliseconds precision for `project_name`, otherwise it failed when 2 backends are started within 1 second.

### CODE REVIEW

**Comment by igor-sirotin** [comment]
@fbarbu15 ~~Can you please check, why there's no python linter check? 🤔 ~~

Oh it's wrong target branch, how was I reviewing 🤦 . Fixed here: 
- https://github.com/status-im/status-go/pull/6230

File: tests-functional/tests/test_init_status_app.py [hunk 1]
```diff
@@ -41,49 +44,45 @@ def test_init_app(self):
...
-        self.assert_file_first_line(
-            logs_dir / "api.log",
-            pattern='"method": "InitializeApplication"',
-            expected=api_logging_enabled,
-        )
+@pytest.mark.rpc
+@pytest.mark.init
+@pytest.mark.parametrize("log_enabled,api_logging_enabled", [(True, True), (False, False)])
+def test_check_logs(log_enabled: bool, api_logging_enabled: bool):
+    data_dir = os.path.join(USER_DIR, "data")
```
Comment (osmaczko) [inline tests-functional/tests/test_init_status_app.py:60]: `USER_DIR` seems somewhat ambiguous. It was unclear to me at first whether it refers to a path that should be set in the context of the host's environment or the container's (status-backend) environment.

File: tests-functional/clients/status_backend.py [hunk 2]
```diff
@@ -154,6 +159,27 @@ def _set_proxy_credentials(self, data):
         data["StatusProxyStageName"] = "test"
         return data
 
+    def load_statusgo_data(self, path: str):
```
Comment (osmaczko) [inline tests-functional/clients/status_backend.py]:
```suggestion
    def extract_container_data(self, path: str):
```
or
```suggestion
    def extract_data(self, path: str):
```

**Review by osmaczko — APPROVED** [review]

**Review by fbarbu15 — APPROVED** [review]

**Review by qfrank — APPROVED** [review]

File: tests-functional/clients/status_backend.py (hunk 2 above)
Comment (igor-sirotin) [reply tests-functional/clients/status_backend.py]: I didn't want to mention `container`, and made this function to be working with host as well, that's why there's a early return:
https://github.com/status-im/status-go/blob/d7f729889aec73d6ebdf8bed60817d690257d331/tests-functional/clients/status_backend.py#L163-L164

Let's make it `extract_data` 👍

File: tests-functional/tests/test_init_status_app.py (hunk 1 above)
Comment (igor-sirotin) [reply tests-functional/tests/test_init_status_app.py:60]: Well, it's the way to work with both docker and local environments.
I agree that it's unobvious, but I guess it's out of the scope of this PR. And I don't really have better ideas how to implement/name this.

**Review by antdanchenko — APPROVED** [review]

**Pull request merged**